bin/ddev-share-e2e

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+PROJECT_HOST=$(ddev wp option get home | tr -d '\r' | awk -F'^http[s]?://' '{print $2}')
+ddev share >/dev/null &
+NGROK_PID=$!
+ngrok-url(){
+    curl -s localhost:4040/api/tunnels | jq -r .tunnels[0].public_url | awk -F'^http[s]?://' '{print $2}'
+}
+wp-replace(){
+  echo "Replacing ${1} with ${2}"
+  ddev wp search-replace "${1}" "${2}" --all-tables --precise --recurse-objects
+}
+
+echo "ngrok started with PID ${NGROK_PID}"
+
+echo "Waiting a few seconds for the service to come up"
+sleep 3
+NGROK_HOST=$(ngrok-url)
+wp-replace "${PROJECT_HOST}" "${NGROK_HOST}"
+echo "Change the baseUrl in tests/e2e/Shared/base-url.js to 'https://${NGROK_HOST}'"
+echo "export const baseUrl = 'https://${NGROK_HOST}';" > tests/e2e/Shared/base-url.js
+echo "Your site is now reachable at https://${NGROK_HOST}"
```

Comment:
How's this different from the other file? At first glance it seems to be doing the same thing, just with the exit handling missing

Reply:
Yeah, the waiting for killing the process in this case caused the action not to be able to move forward. One step goes after the other, only jobs can run in parallel. And in any case we would create a new instance every time, so rolling back to the project urls is not needed. But another approach could be to extract this rollback to another command instead of waiting and so we could use the same for local and action... not sure what is better

Reply:
Perhaps a parameter could be added that optionally skips the exit handling